packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -143,6 +144,7 @@ export class LedgerHDWalletInfo
   readonly _supportsETHInfo = true;
   readonly _supportsThorchainInfo = true;
   readonly _supportsCosmosInfo = true;
+  readonly _supportsSolanaInfo = true;
```

Comment:
implements core.SolanaWalletInfo for LedgerHDWalletInfo

Reply:
170b4ee2bc0222ffeb4517369ee02a1c7904feca